content/docs/integrations/github-org/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Configuring the GitHub Organization
+lastUpdated: '2021-03-19T12:14:39.0Z'
+description: How to configure backstage to read teams from github
+---
+
+## Introduction
+
+You may want to load you organization team structure from github teams.
+
+This page describes how to do that.
+
+## Prerequisites
+
+- You will need a git repository for the configuration of Backstage. In the examples
+  below, I am calling this backstage-configuration
```

Comment:
```suggestion
  below, I am calling this backstage-configuration.
```

BTW, as far as I can see, you don't actually refer to this configuration anywhere in the file?

Reply:
removed the reference